hooks/pre-commit

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from future import standard_library
+standard_library.install_aliases()
+
+import commands
+import os
+import sys
+import urllib.request, urllib.parse, urllib.error
+from distutils.version import LooseVersion
+import re
+
+pattern = re.compile(r'script_version["\s:=]+(\d+\.\d+)')
+version_dict = {}
+versionfile = urllib.request.urlopen(
+    "https://raw.githubusercontent.com/henrykironde/retriever/1f18008838cecf7f8ac208d57cd2a578e50e1c2c/version.txt")
+versionfile.readline()
+
+for lines in versionfile:
+    key_values = lines.split(",")
+    version_dict[key_values[0]] = key_values[1]
+
+
+def get_script_version(file_name):
+    file_name = os.path.split(file_name)[1]
+    return LooseVersion(version_dict[file_name])
+
+
+# check if script directory changes
+status, output = commands.getstatusoutput("git diff --name-only origin/script2 scripts")
```

Comment:
This was unchanged from before. I was unsure what we were trying to do with this. Were we trying to see if the scripts had been moved to a different directory?